### ISSUE TITLE
Initiate navigate before delete

### ISSUE THREAD

**Issue by twMat** [Jermolene/TiddlyWiki5#3947]
Clicking "delete tiddler" causes a moment of freeze before navigation to next tiddler takes place. This is one of those typical "wait, but keep on staring" annoyances for web pages.

I wish for a test to see if it possible AND would make sense with navigation before initiating the deletion. It may be that this would cause a second scrolling when the tiddler actually disappears, or maybe the freeze would halt the scrolling mid-process for a choppy impression, I don't know.

**Comment by pmario** [comment]
Did you test this on tiddlywiki.com. .... I can't see any delay. If you change animation time to 2 secondes, you can see, that it is done automatically. 

How does your setup look like?

**Comment by twMat** [comment]
Sorry, poorly phrased OP; The delay is between when delete is clicked and the actual deletion. Not after the deletion and before scrolling. This is most easily noted when setting animation to 0.

**Comment by pmario** [comment]
Which OS? Which Browser? Which Story-View do you use?
I can't see any delay at all

**Comment by twMat** [comment]
That's interesting. Maybe you have a more powerful computer than mine?
Win10 home, Chrome. Default storyview. ... 4GB, 1,1Ghz, likely some cheap graphics card.
But I'm not talking about a *big* delay. Maybe one second? Still, as noted, it is one of those "wait" moments so if it can easily be avoided...

**Comment by pmario** [comment]
Hmmm. 1 second is a "big" delay. Any browser AddOns active, that may delay UI response times?

**Comment by twMat** [comment]
Yeah, many browser add ons. No idea if it affects anything. I'll look into this within a few days.

**Comment by twMat** [comment]
BTW, thanks for your input on the matter.

**Comment by AnthonyMuscio** [comment]
twMat, A Delete should trigger auto save if on, so the delay includes saving the whole wiki (when a single file wiki) - turn off autosave and measure again.

Tony
